Header header
float64[] r
float64[] rho_max
float64[] gamma_0
float64[] gamma_inf
float64[] l